docs/en/observability/apm/known-issues.asciidoc

```diff
@@ -37,6 +37,26 @@ Failed installing package [apm] due to error: [ResponseError: mapper_parsing_exc
 // Link to fix?
 A fix was released in 8.11.2: https://github.com/elastic/kibana/pull/171712[elastic/kibana#171712].
 
+*traces-apm@custom ingest pipeline applied to certain data streams unintentionally* +
+_APM Server versions: 8.12.0_ +
+
+// Describe the conditions in which this issue occurs
+If you're using the Elastic APM Server v8.12.0,
+// Describe the behavior of the issue
+the `traces-apm@custom` ingest pipeline is now additionally applied to data streams `traces-apm.sampled-*`
+and `traces-apm.rum-*`, and applied twice for `traces-apm-*`. This bug impacts users with a non-empty `traces-apm@custom` ingest pipeline.
+
+If you rely on this unintended behavior in 8.12.0, please rename your pipeline to `traces-apm.package@custom` to preserve this behavior in 8.12.1.
```

Comment:
```suggestion
If you rely on this unintended behavior in 8.12.0, please rename your pipeline to `traces-apm.package@custom` to preserve this behavior in later versions.
```
^ will this be clearer that it applies to 8.12.1+? or is it already implied?

Reply:
The fix actually ended up using `.integration` and not `package` as suffix, see https://github.com/elastic/package-spec/pull/704 and https://github.com/elastic/kibana/pull/175448/files.

Reply:
good catch! I copied from PR description without noticing it